netty/src/main/java/io/grpc/transport/netty/NettyClientStream.java

```diff
@@ -128,7 +129,8 @@ protected void sendFrame(WritableBuffer frame, boolean endOfStream, boolean flus
     if (numBytes > 0) {
       // Add the bytes to outbound flow control.
       onSendingBytes(numBytes);
-      channel.write(new SendGrpcFrameCommand(this, bytebuf, endOfStream)).addListener(
+      channelWriter.enqueue(
```

Comment:
Its definitely not. I've kept the baseline flush suppression and add a 'flush' param to the enqueue methods to accomodate this. Ill add the remaining explicit flush suppression in a follow up